acceptance/openstack/loadbalancer/v2/flavorprofiles_test.go

```diff
@@ -0,0 +1,33 @@
+//go:build acceptance || networking || loadbalancer || flavorprofiles
+// +build acceptance networking loadbalancer flavorprofiles
+
+package v2
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+	"github.com/gophercloud/gophercloud/openstack/loadbalancer/v2/flavorprofiles"
+)
+
+func TestFlavorProfilesList(t *testing.T) {
```

Comment:
The acceptance tests are extremely useful to see if the code works on all supported versions of OpenStack, against real environment.

Ideally they should test all the API calls: list, create, update, and delete.

Reply:
I made a CRUD test